app/services/pii/password_encryptor.rb

```diff
@@ -6,7 +6,7 @@ def initialize
     end
 
     def encrypt(plaintext, user_access_key)
-      encrypted_key_maker.make(user_access_key)
+      encrypted_key_maker.make(user_access_key) unless user_access_key.made?
```

Comment:
Reek doesn't like the duck typing (at least, using `respond_to` -- I'm not sure if there are better ways to check).

https://github.com/troessner/reek/blob/master/docs/Manual-Dispatch.md